rtbkit/core/banker/go_account.h

```diff
@@ -28,8 +28,8 @@ struct GoBaseAccount {
 };
 
 struct GoRouterAccount : public GoBaseAccount {
-    std::atomic<int64_t> rate;
-    std::atomic<int64_t> balance;
+    Amount rate;
+    Amount balance;
```

Comment:
So you went from atomic to not atomic; are you sure that no code is affected by this? Do not that std::atomic is not just about atomic accesses and writes and also about ordering.